src/AppBundle/Utils/Markdown.php

```diff
@@ -23,15 +23,19 @@ class Markdown
 {
     private $parser;
 
+    private $purifier;
+
     public function __construct()
     {
         $this->parser = new \Parsedown();
+        $this->purifier = new \HTMLPurifier();
```

Comment:
Anyway, I thought it'll be clearer to get separate service for purifier.

Do you think it'll be a bit overhead for available services?